lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -53,6 +50,14 @@ def build(namespace:, context:, definition:, logger:, statsd_tags:, crd: nil)
         end
       end
 
+      def class_for_kind(kind)
+        if KubernetesDeploy.const_defined?(kind)
+          KubernetesDeploy.const_get(kind) # rubocop:disable Sorbet/ConstantsFromStrings
```

Comment:
> Sorbet/ConstantsFromStrings: Don't use `const_get`, it make the code harder to understand, less editor-friendly and impossible to analyze. Replace `const_get` with a case/when or a hash.

That suggestion seems out of scope for this PR, so I chose to disable the cop.

Reply:
> That suggestion seems out of scope for this PR

I agree